sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/CodeSnippetsAysnc.java

```diff
@@ -0,0 +1,146 @@
+package com.azure.data.tables;
+
+
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+
+
+public class CodeSnippetsAysnc {
```

Comment:
These shouldn't be in the "src/main/java" folder. The "main" folder is for product code.
These should be in "src/samples/java". https://azure.github.io/azure-sdk/java_documentation.html#code-samples
Since these are snippets dealing with the "TableServiceAsyncClient", this file can be called "TableServiceAsyncClientCodeSamples".

Here's an example.
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs